scripts/run-local-perf-test.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+SCRIPT_BASE="$(cd -P "$(dirname "$0")" && pwd)"
+
+# Usage:
+# run-local-smoke-test.sh
+
+# Either have these environment variables set, or change the script below (don't commit that!)
+
+# export PORT_CLIENT_ID=""
+# export PORT_CLIENT_SECRET=""
+# export PORT_BASE_URL=http://localhost:3000
```

Comment:
missing all extra params that can be set here

Reply:
Added